www/breakdownTimeline.php

```diff
@@ -31,52 +31,63 @@
             if (isset($processing)) {
                 arsort($processing);
                 $mapping = array('EvaluateScript' => 'Scripting',
-                               'v8.compile' => 'Scripting',
+                               'EventDispatch' => 'Scripting',
+                               'FireAnimationFrame' => 'Scripting',
+                               'FireAnimationFrame' => 'Scripting',
                                'FunctionCall' => 'Scripting',
                                'GCEvent' => 'Scripting',
+                               'Heap::collectGarbage' => 'Scripting',
+                               'HitTest' => 'Scripting',
+                               'IntersectionObserverController::computeIntersections' => 'Scripting',
+                               'MajorGC' => 'Scripting',
+                               'MinorGC' => 'Scripting',
+                               'ThreadState::completeSweep' => 'Scripting',
+                               'ThreadState::performIdleLazySweep' => 'Scripting',
                                'TimerFire' => 'Scripting',
-                               'EventDispatch' => 'Scripting',
                                'TimerInstall' => 'Scripting',
                                'TimerRemove' => 'Scripting',
+                               'v8.compile' => 'Scripting',
```

Comment:
this is taken care of later in `stripos($type, 'v8.')`, right?

Reply:
Yep.  Left it just for legacy reasons but happy to just remove it